1/styles/scss/_media-tablet.scss

```diff
@@ -0,0 +1,118 @@
+@media screen and (max-width: 992px) {
+  .d-md-block{
+    display: block;
+  }
+  .d-md-none{
+    display: none;
+  }
+  .logo + div{
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin-right: 40px;
+  }
+
+  .accordion-item-content .source{
+    margin: 20px 0;
+  }
+
+  .header-menu{
+    z-index: 5;
+    ul{
+      flex-direction: column;
+      position: absolute;
+    }
+    .active::after{
+      background: $accent;
+      left: 48%;
```

Comment:
do not use magic numbers, center this block another way, you need to use left 50% and transform the element according itself

Reply:
done